src/messages/DatePicker/flatpickr-plugins/customElements.ts

```diff
@@ -97,6 +136,8 @@ function customElements(pluginConfig: Config): Plugin {
 				() => {
 					fp?.loadedPlugins?.push("customElements");
 				},
+				setMonthSelectAlly,
```

Comment:
I would suggest to move these two new functions above the line 136. I remember that the order of this array is relevant on rendering the datepicker and the function in line 137 should be the last one.

Reply:
done, thanks